resources/views/layouts/app.blade.php

```diff
@@ -32,19 +32,20 @@
 
                 <div class="collapse navbar-collapse" id="navbarSupportedContent">
                     <!-- Left Side Of Navbar -->
-                    <ul class="navbar-nav mr-auto">
-                        <li><a class="navbar-brand" href="{{ route('pages.index') }}"> Pages </a> </li>
-                        <li><a class="navbar-brand" href="{{ route('menus.index') }}"> Menus </a></li>
-                    </ul>
-
+                    @if (Auth::check())
+                        <ul class="navbar-nav mr-auto">
+                            <li><a class="navbar-brand" href="{{ route('pages.index') }}"> Pages </a> </li>
+                            <li><a class="navbar-brand" href="{{ route('menus.index') }}"> Menus </a></li>
+                        </ul>
+                    @endif
                     <!-- Right Side Of Navbar -->
                     <ul class="navbar-nav ml-auto">
                         <!-- Authentication Links -->
                         @guest
                             <li class="nav-item">
                                 <a class="nav-link" href="{{ route('login') }}">{{ __('Login') }}</a>
                             </li>
-                            @if (Route::has('register'))
+                            @if (Auth::check())
```

Comment:
This has not been resloved. `Auth::check()` will always be `false` inside `@guest`, as per the defenition of the blade guest directive, the user is NOT logged in

Reply:
I did it 👍